src/index.html

```diff
@@ -7,8 +7,69 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header
+      class="header"
+    >
+      <a href="#">
+        <img
+          src="images/logo.png"
+         alt="Logo"
+         class="picture"
+       >
+      </a>
+      <nav class="nav">
+        <ul class="nav__item">
+          <li class="nav__list">
+            <a href="#"
+              class="is-active nav__links"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__list">
```

Comment:
add one empty line between same level elements
```suggestion
          </li>
          
          <li class="nav__list">
```